packages/workspace/src/generators/move/lib/update-eslintrc-json.ts

```diff
@@ -41,28 +23,14 @@ export function updateEslintrcJson(
   schema: NormalizedSchema,
   project: ProjectConfiguration
 ) {
+  tree['recordedChanges'];
   const eslintRcPath = join(schema.relativeToRootDestination, '.eslintrc.json');
-
   if (!tree.exists(eslintRcPath)) {
     // no .eslintrc found. nothing to do
     return;
   }
 
-  const offset = offsetFromRoot(schema.relativeToRootDestination);
-
   updateJson<PartialEsLintRcJson>(tree, eslintRcPath, (eslintRcJson) => {
-    if (typeof eslintRcJson.extends === 'string') {
```

Comment:
I feel like we should keep it in here and take it out of `updateRootProjectFIles` since this is more specific.

And we don't need 2 places that both update the `eslintrc.json

Reply:
That was my initial take, but the other one is more generic. I would have to explicitly exclude it from root files.

Reply:
I'll move the logic here then